bip-p2qrh.mediawiki

```diff
@@ -0,0 +1,598 @@
+<pre>
+  BIP: ?
+  Title: QuBit: SegWit version 3 spending rules (P2QRH)
+  Layer: <Consensus (soft fork)
+  Author: Hunter Beast <hunter@surmount.systems>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-TBD
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-06-08
```

Comment:
Please update the BIP number and filename, and the created date to 2024-12-18 (date of BIP number assignment) and add a corresponding BIP draft entry to the repo root README file.

Reply:
Done!